### PR TITLE
Prepare for 1.6.0 Release

### PR DESCRIPTION
**Pull request by cbgbt**
**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
+# v1.6.0 (2022-02-07)
+
+## Deprecation Notice
+
+The Kubernetes 1.18 variant, `aws-k8s-1.18`, will lose support in March, 2022.
```
Comment (bcressey) [inline CHANGELOG.md]: nit:
```suggestion
The Kubernetes 1.18 variant, `aws-k8s-1.18`, will lose support in March 2022.
```

**Review by bcressey — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+[#1859]: https://github.com/bottlerocket-os/bottlerocket/pull/1859
+[#1860]: https://github.com/bottlerocket-os/bottlerocket/pull/1860
+[#1861]: https://github.com/bottlerocket-os/bottlerocket/pull/1861
+[#1862]: https://github.com/bottlerocket-os/bottlerocket/pull/1862
+[#1867]: https://github.com/bottlerocket-os/bottlerocket/pull/1867
+[#1883]: https://github.com/bottlerocket-os/bottlerocket/pull/1883
+[#1889]: https://github.com/bottlerocket-os/bottlerocket/pull/1889
+[#1894]: https://github.com/bottlerocket-os/bottlerocket/pull/1894
+[#1896]: https://github.com/bottlerocket-os/bottlerocket/pull/1896
+[#1897]: https://github.com/bottlerocket-os/bottlerocket/pull/1897
```
Comment (etungsten) [inline CHANGELOG.md]: Not sure if intended. This links to an issue, the PR that addresses this issue is https://github.com/bottlerocket-os/bottlerocket/pull/1906

**Comment by cbgbt** [comment]
Force pushed to add missing PRs, fix grammar error.

**Review by etungsten — APPROVED** [review]

**Comment by cbgbt** [comment]
Fixed a link that mistakenly pointed to an issue, rather than the PR that resolved it.

**Review by zmrow — APPROVED** [review]
:kick_scooter:

**Review by etungsten — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**